packages/governance/src/contractGovernance/paramManager.js

```diff
@@ -231,7 +231,7 @@ const makeParamManagerBuilder = zoe => {
      *
      * @param {SetInvitationParam} param0
      */
-    const setInvitation = async ([newInvitation, amount]) => {
+    const setInvitation = ([newInvitation, amount]) => {
```

Comment:
Removing the `async` converts some potential rejections into potential throws. Is this consistent with existing callers?

Reply:
> Removing the async converts some potential rejections into potential throws. Is this consistent with existing callers?

I planned to rely on unit and types testing to confirm that. But now I'm thinking to [eschew any semantic changes in this PR](https://github.com/Agoric/agoric-sdk/pull/5410#discussion_r878746786).